cli.js

```diff
@@ -48,6 +49,7 @@ var cli = meow([
 	'  --tap, -t        Generate TAP output',
 	'  --verbose, -v    Enable verbose output',
 	'  --no-cache       Disable the transpiler cache',
+	'  --watch, -w      Run tests when files change',
```

Comment:
This should be commented out. We decided to not yet expose this. https://github.com/sindresorhus/ava/issues/70#issuecomment-173746486